Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -69,5 +74,25 @@ def commit(self):
             distances = inv(distances)
         self.send("Distances", distances)
 
+    def send_report(self):
+        items = OrderedDict()
+        norm, normopt = self.normalization_method, self.normalization_options
+        inv, invopt = self.inversion_method, self.inversion_options
+        if norm == 0:
```

Comment:
Independently of @kernc, I wrote basically the same code in https://github.com/biolab/orange3/pull/1100#issuecomment-195727741. I wrote that you can keep your code, but if I'm not the only one who prefers not duplicating code, then please change it.

I suspect that you got the error since you misplaced the parentheses. @kernc's code looks OK (and he probably tested it, too. :)).

@kernc, can #1100 be closed to reduce the mess?